KvNGCzA/Hello-Books-BE
# #166214087 adds prettier and lint stage to the repo

#### Description
It sets up prettier and lint-stage to the repository. Prettier is added in collaboration with eslint
#### How can this be Tested?
* pull the branch to your local 
* use `npm install` to download all necessary packages
* download and enable prettier as an extension in your editor
* create a .js file and observe the rules being applied as you write codes in the file
* on `git commit` all staged files are checked to see they follow the specified rules
#### Checklist:
N/A
#### pivotal tracker story
https://www.pivotaltracker.com/story/show/166214087
